frontend/utility/MissingFilesModel.cpp

```diff
@@ -160,86 +154,128 @@ Qt::ItemFlags MissingFilesModel::flags(const QModelIndex &index) const
 	return flags;
 }
 
-void MissingFilesModel::fileCheckLoop(QList<MissingFileEntry> files, QString path, bool skipPrompt)
+void MissingFilesModel::findAllFilesInPath(QList<MissingFileEntry> files, QString path, bool skipPrompt)
```

Comment:
```suggestion
void MissingFilesModel::findAllFilesInPath(const QList<MissingFileEntry> &files, const QString &path, bool skipPrompt)
```

As this function serves more or less as just a "forwarder", it needs to match the argument types, otherwise both get copied for this function and then those copies' references are passed into `fileCheckLoop`.